### ISSUE TITLE
Shell-fm doesn't work inside screen

### ISSUE THREAD

**Issue by Daenyth** [jkramer/shell-fm#4]
I've also tried setting TERM to rxvt, that didn't work either. I'm on Arch Linux i686. It works outside of screen. Hitting enter gives the "Sorry, couldn't set station to..." message with the junk after it. Hitting the "r" bind spits out the junk again.

daenyth@Muspelheimr ~ $ shell-fm
Shell.FM v0.7, (C) 2006-2009 by Jonas Kramer
Published under the terms of the GNU General Public License (GPL).

Press ? for help.

Password: 
radio url> ����������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������

**Comment by jkramer** [comment]
I use shell-fm in screen all day, it works just fine. Are you using some special settings, like for the screen title?

**Comment by Daenyth** [comment]
Don't think so... Here's my screenrc:
startup_message off
defutf8 on
defscrollback 10000
hardstatus off

Commenting out the hardstatus line in my screenrc seemed to make it go away. I don't know why it would break it in the first place.

**Comment by jkramer** [comment]
I mean in shell-fm.rc.

**Comment by Daenyth** [comment]
I only set username in shell-fm.rc.

Removing my hardstatus line from screenrc makes the issue go away though. That's really weird.

**Comment by jkramer** [comment]
Closing this issue as it seems to be caused by the screen configuration.

**Comment by Daenyth** [comment]
I don't really think that's accurate... It's being triggered by it surely, but no other program has any issue with the setting, which I've had for several years. That leads me to believe the code here is doing something wrong

**Comment by jkramer** [comment]
Ok, re-opened. I'll try to reproduce it.

**Comment by Daenyth** [comment]
Thanks :)

**Comment by jkramer** [comment]
I've tested with the screenrc you've posted above, trying both hardstatus on and off. It worked fine in both cases. Do you have additional settings?

**Comment by Daenyth** [comment]
Not really... Only other info I might give is that I typically start the screen at a vc before I start X, and then reattach from urxvt.

**Comment by jkramer** [comment]
Still works fine. Maybe a urxvt/tty setting?

**Comment by Daenyth** [comment]
My Xdefaults are here, but nothing unusual really... http://github.com/Daenyth/dotfiles

**Comment by jkramer** [comment]
Can you give me a hexdump -C of the garbage output you're getting? Not sure if the characters you posted above have been converted by github.

**Comment by Daenyth** [comment]
Will do.

Also, my shell-fm binary is linked against:

```
gcc-libs 4.5.0-1
glibc 2.11.1-3
libao 1.0.0-1
libmad 0.15.1b-4
taglib 1.6.3-1
zlib 1.2.5-1
```

And here's the output from a portion of the text:

```
daenyth@Muspelheimr ~ $ hexdump -C 
 ������������������������������������������������������������������������������������
00000000  20 ef bf bd ef bf bd ef  bf bd ef bf bd ef bf bd  | ...............|
00000010  ef bf bd ef bf bd ef bf  bd ef bf bd ef bf bd ef  |................|
00000020  bf bd ef bf bd ef bf bd  ef bf bd ef bf bd ef bf  |................|
00000030  bd ef bf bd ef bf bd ef  bf bd ef bf bd ef bf bd  |................|
00000040  ef bf bd ef bf bd ef bf  bd ef bf bd ef bf bd ef  |................|
00000050  bf bd ef bf bd ef bf bd  ef bf bd ef bf bd ef bf  |................|
00000060  bd ef bf bd ef bf bd ef  bf bd ef bf bd ef bf bd  |................|
00000070  ef bf bd ef bf bd ef bf  bd ef bf bd ef bf bd ef  |................|
00000080  bf bd ef bf bd ef bf bd  ef bf bd ef bf bd ef bf  |................|
00000090  bd ef bf bd ef bf bd ef  bf bd ef bf bd ef bf bd  |................|
000000a0  ef bf bd ef bf bd ef bf  bd ef bf bd ef bf bd ef  |................|
000000b0  bf bd ef bf bd ef bf bd  ef bf bd ef bf bd ef bf  |................|
000000c0  bd ef bf bd ef bf bd ef  bf bd ef bf bd ef bf bd  |................|
000000d0  ef bf bd ef bf bd ef bf  bd ef bf bd ef bf bd ef  |................|
000000e0  bf bd ef bf bd ef bf bd  ef bf bd ef bf bd ef bf  |................|
000000f0  bd ef bf bd ef bf bd ef  bf bd ef bf bd 0a        |..............|
000000fe
```
